CMakeLists.txt

```diff
@@ -445,7 +449,46 @@ if(AVIF_CODEC_AOM)
     message(STATUS "libavif: Codec enabled: aom (${AVIF_CODEC_AOM_ENCODE_DECODE_CONFIG})")
 endif()
 
-if(NOT AVIF_CODEC_AOM AND NOT AVIF_CODEC_DAV1D AND NOT AVIF_CODEC_LIBGAV1)
+if(AVIF_CODEC_AVM)
+    message(WARNING "libavif: AV2 support with avm is experimental. Only use for testing.")
+
+    # The avm repository is a fork of aom and inherited a lot of folders, files and build artifacts named the same way.
+    # Having both dependencies at the same time generates conflicts in includes, binary lookups etc.
+    if(AVIF_CODEC_AOM)
```

Comment:
I found that AVM and SVT-AV1 both define a global function `compute_distortion`, so if we link with both AVM and SVT-AV1 as static libraries, we will get a "multiple definition of `compute_distortion'" linker error.

This can be avoided by declaring `compute_distortion` as a file-static function in AVM or SVT-AV1 (or both).

Reply:
SVT-AV1 fixed this issue in https://gitlab.com/AOMediaCodec/SVT-AV1/-/commit/61ff779ea2f2b908cce00cec4cd304a06fc88a93?page=2, and it is part of their v1.5.0 release. I will update libavif to check out SVT-AV1 v1.5.0 in https://github.com/AOMediaCodec/libavif/pull/1377. Then we will be able to modify .github/workflows/ci-unix-static-av2.yml to build SVT-AV1.

Reply:
> Then we will be able to modify .github/workflows/ci-unix-static-av2.yml to build SVT-AV1.

Done in https://github.com/AOMediaCodec/libavif/pull/1395.